build/yamls/base/crds.yml

```diff
@@ -270,7 +270,7 @@ spec:
                       prefixLength:
                         type: integer
                       vlan:
-                        type: string
+                        type: integer
```

Comment:
You can put a limitation here: `maximum: 4094`

Reply:
Thanks. Added `minimum` and `maximum` in CRD.